### PR TITLE
Initial Draft of Homepage

### PR DESCRIPTION
**Pull request by davidsmejia**
## Issue Number

Adds the homepage and some other components needed for downstream PR.

Static version of the account section is up next.

### CODE REVIEW

**Comment by davidsmejia** [comment]
Screen Shot 2020-08-18 at 4 42 25 PM

File: client/src/components/AccountLayout.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    },
+    {
+      text: 'Teams',
+      href: '/account/teams',
+      notifications: 0
+    },
+    {
+      text: 'Notifications',
+      href: '/account/notifications',
+      notifications: 4
```
Comment (kurtwheeler) [inline client/src/components/AccountLayout.js:33]: Is having 4 notifications hard coded in?

Comment (davidsmejia) [reply client/src/components/AccountLayout.js:33]: yeah needs to be integrated still that will be done with account notifications section integration

File: client/src/pages/index.js [hunk 2]
```diff
@@ -23,7 +313,7 @@ Home.getInitialProps = async ({ req, query }) => {
     return {}
   }
 
-  let queryJSON = {}
+  let queryJSON = { email: 'this@willchange.com' }
```
Comment (kurtwheeler) [inline client/src/pages/index.js:316]: When?

Comment (davidsmejia) [reply client/src/pages/index.js:316]: this week

**Pull request merged**